src/coreclr/Directory.Build.props

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
   <Import Project="..\..\Directory.Build.props" />
 
+  <!-- foo -->
```

Comment:
Guessing you didn't mean to add this :smile:

Reply:
I did, to trigger coreclr and mono tests to increase the probability of hitting the issue. I added it as a separate commit though. Will remove it before merging of course.